### PR TITLE
Modal Component

### PR DESCRIPTION
**Pull request by Tanay-J**
This PR adds a Modal component to inkUI component library.

**Demo**

https://user-images.githubusercontent.com/33786522/160145314-cc388f1b-cc17-4ab9-b479-de4d499094bf.mp4

### CODE REVIEW

File: components/modal.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.modal-wrapper {
+  z-index: 1;
+  background-color: rgba(0, 0, 0, .5);
```
Comment (Mohitk391) [inline components/modal.css:3]: Suggestion: You can replace rgba with color variable to increase reusability and redundancy of the color.

**Review by Mohitk391 — APPROVED** [review]
Appreciation: Your UI looks simple, awesome. Your code is also clear, clean and readable. Loved it..You can merge the branch...!!!!

File: components/modal.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```
Comment (adith-a-danthi) [inline components/modal.css:8]: You can use the shorthand property `inset` for position.

**Review by adith-a-danthi — COMMENTED** [review]
Clean code and nice minimal UI.
Looks good to merge

File: components/modal.css (hunk 2 above)
Comment (Tanay-J) [reply components/modal.css:8]: Thanks for pointing it out, 'll update before merging.

File: components/modal.css (hunk 1 above)
Comment (Tanay-J) [reply components/modal.css:3]: This is one-off usage of this color, hence directly used rgba value.

**Pull request merged**